packages/xstate-wallet/src/chain.ts

```diff
@@ -151,7 +152,7 @@ export class FakeChain implements Chain {
   }
 
   public depositSync(channelId: string, expectedHeld: string, amount: string) {
-    const current = (this.channelStatus[channelId] || {}).amount || bigNumberify(0);
+    const current = (this.channelStatus[channelId] || {}).amount || BigNumber.from(0);
```

Comment:
In general I like to use `Zero` instead of `BigNumber.from(0)`. We should probably stick to one convention.